src/test/java/teammates/test/cases/ui/browsertests/StudentHomePageUiTest.java

```diff
@@ -67,14 +67,14 @@ private void testContentAndLogin() throws Exception {
                                                       .loginAsStudent(unregUserId, unregPassword);
 
         // this test uses the accounts from test.properties
-
-        // This is the full HTML verification for Student Home Page, the rest can all be verifyMainHtml
-        studentHome.verifyHtml("/studentHomeHTMLEmpty.html");
+        studentHome.verifyHtmlMainContent("/studentHomeHTMLEmpty.html");
```

Comment:
we can consider providing reason here as to why this one is MainContent and the full html check comes later. Even if we don't spell the reason out in full, we should at least caution the developer from changing this. Also provide a quick reference to this pr/issue.